package.json

```diff
@@ -16,13 +16,14 @@
     "postpublish": "pinst --enable"
   },
   "dependencies": {
-    "@graphql-codegen/plugin-helpers": "^5.0.1",
+    "@graphql-codegen/plugin-helpers": "^6.0.0",
+    "@graphql-codegen/visitor-plugin-common": "^6.0.0",
     "@homebound/activesupport": "^1.9.0",
     "change-case": "^4.1.2",
-    "ts-poet": "^6.6.0"
+    "ts-poet": "^6.12.0"
```

Comment:
I need the newer ts-poet to handle the subpath imports, as well. The 6.6.0 version didn't work properly.